src/block_disagg/block_disagg_ckpt.c

```diff
@@ -87,6 +87,29 @@ __wt_block_disagg_checkpoint(WT_BM *bm, WT_SESSION_IMPL *session, WT_ITEM *root_
     return (0);
 }
 
+/*
+ * __block_disagg_update_shared_metadata --
+ *     Update the shared metadata.
+ */
+static int
+__block_disagg_update_shared_metadata(
+  WT_BM *bm, WT_SESSION_IMPL *session, const char *key, const char *value)
+{
+    WT_CURSOR *cursor;
+    WT_DECL_RET;
+    const char *cfg[] = {WT_CONFIG_BASE(session, WT_SESSION_open_cursor), "overwrite", NULL, NULL};
+
+    WT_UNUSED(bm);
+
+    WT_ERR(__wt_open_cursor(session, WT_DISAGG_METADATA_URI, NULL, cfg, &cursor));
+    cursor->set_key(cursor, key);
+    cursor->set_value(cursor, value);
+    WT_ERR(cursor->insert(cursor));
+
+err:
```

Comment:
cursor should be closed here?

Reply:
Good catch! Let me fix that.

Reply:
Fixed in [595f67f](https://github.com/wiredtiger/wiredtiger/pull/11211/commits/595f67f13c1042cd5b286e63252c89647b9a6e07).